test/e2e/node/pods.go

```diff
@@ -298,7 +298,49 @@ var _ = SIGDescribe("Pods Extended", func() {
 				}
 			}
 		})
+
+		ginkgo.It("evicted pods should be terminal", func() {
+			ginkgo.By("creating the pod that should be evicted")
+
+			name := "pod-should-be-evicted" + string(uuid.NewUUID())
+			image := imageutils.GetE2EImage(imageutils.BusyBox)
+			pod := &v1.Pod{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: name,
+				},
+				Spec: v1.PodSpec{
+					RestartPolicy: v1.RestartPolicyOnFailure,
+					Containers: []v1.Container{
+						{
+							Name:  "bar",
+							Image: image,
+							Command: []string{
+								"/bin/sh", "-c", "sleep 10; fallocate -l 10M file; sleep 10000",
+							},
+							Resources: v1.ResourceRequirements{
+								Limits: v1.ResourceList{
+									"ephemeral-storage": resource.MustParse("5Mi"),
```

Comment:
@SergeyKanzhelev I'm not familiar with this area, I've used it from the reproducer used in the reported issue, is this right or is there a better way to create an evicted pod?

Reply:
If there is, we should potentially create a test function others can use (it’s extremely useful to be able to create evicted pods we know a kubelet will reject).

Reply:
I will leave the test as is, it is exactly the same as the reproducer, so it will serve as regression for this bug